gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/jsr223/GremlinScriptCheckerTest.java

```diff
@@ -192,12 +194,41 @@ public void shouldIdentifyMultipleRequestIds() {
                 getRequestId().get());
     }
 
+    @Test
```

Comment:
please include test variations for `Tokens` styled naming when that's added given the previous comments.